scripts/buildCss.js

```diff
@@ -0,0 +1,41 @@
+// core packages
+const fs = require('fs'),
+      path = require('path')
+
+// npm packages
+const simpleIcons = require('simple-icons')
+
+
+const basePath = path.join(__dirname, '..')
+
+const attributedIcons = Object.values(simpleIcons).map(icon => {
+	return {
+		color: icon.hex,
+		cssClass: icon.title
+			.toLowerCase()
+			.replace(/[ !’]/g, '')
+			.replace(/-/g, '_') // hyphens not supported in ligatures
+			.replace(/\./g, 'dot') // dot not supported in css class names
+			.replace(/\+/g, 'plus') // plus not supported in css class names
+			.replace(/&/g, 'and') // ampersand not supported in css class names
+		,
+		ligature: icon.title
+			.replace(/[ !’]/g, '')
+			.replace(/-/g, '_') // hyphens not supported in ligatures
+	}
+})
+
+fs.readFile(path.join(basePath, 'src', 'css', 'base.css'), (err, baseCss) => {
+	if(err) throw err
+
+	attributedIcons.forEach(icon => {
+		baseCss += '\n'
+		+   `.simpleicons-${icon.cssClass}::before { content: '${icon.ligature}'; }\n`
```

Comment:
To reduce the package size we should consider removing the spaces and newlines here, or add a minimiser at the end.

Reply:
Oh yeah, that makes sense.
I think building the "pretty" version for debugging (excluded from the package) and distributing a minified version instead would make sense.

Reply:
I'm fine either way 😉 In that case a minimiser at the end that runs based on an argument/environment variable is probably the way to go, agreed?